test/problems/second_quantization/molecular/test_fermionic_op_builder.py

```diff
@@ -0,0 +1,48 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Tests Fermionic Operator builder."""
+from test import QiskitNatureTestCase
+from qiskit_nature.operators import FermionicOp
+from qiskit_nature.problems.second_quantization.molecular import fermionic_op_builder
+from qiskit_nature.drivers import HDF5Driver
+
+
+# TODO add more detailed tests
+class TestFermionicOperatorBuilder(QiskitNatureTestCase):
```

Comment:
I guess we have no way of doing some exact diagonalization on the operator to get a eigenvalue(s) that might help in testing to determine if it appears correct.

Reply:
Do you mean as an alternative to the explicit list checking done here or in addition to that?
I think for now, the way it is done is fine, but this is definitely something we can keep in mind for the future :+1:

Reply:
I mean it could be used perhaps for the unit test but was thinking more for something on operator - hence if it were there it could be used by the unit test too.